python/plugins/processing/algs/gdal/GridAverage.py

```diff
@@ -180,6 +182,9 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
         arguments.append('-of')
         arguments.append(output_format)
 
+        if input_details.open_options:
```

Comment:
Support for open options in this tool was added in GDAL 3.7. Maybe we should check GDAL version here to support old platforms?